scippy/src/scippy/plot.py

```diff
@@ -303,30 +307,61 @@ def plot_image(input_data, name=None, axes=None, contours=False, cb=None,
     layout = dict(
         xaxis=dict(title=axis_label(xcoord)),
         yaxis=dict(title=axis_label(ycoord)),
-        height=default["height"]
+        height=default["height"],
+        width=default["width"]
     )
+    if input_data.has_variances:
+        layout["width"] *= 0.5
+        layout["height"] = 0.8 * layout["width"]
 
     if plot:
-        z = input_data.values
+        values = input_data.values
         # Check if dimensions of arrays agree, if not, plot the transpose
         if (zlabs[0] == xlabs[0]) and (zlabs[1] == ylabs[0]):
-            z = z.T
+            values = values.T
         # Apply colorbar parameters
         if cbar["log"]:
             with np.errstate(invalid="ignore", divide="ignore"):
-                z = np.log10(z)
+                values = np.log10(values)
         if cbar["min"] is None:
-            cbar["min"] = np.amin(z[np.where(np.isfinite(z))])
+            cbar["min"] = np.amin(values[np.where(np.isfinite(values))])
         if cbar["max"] is None:
-            cbar["max"] = np.amax(z[np.where(np.isfinite(z))])
-        imview = ImageViewer(xe, xc, ye, yc, z, resolution, cbar,
-                             plot_type, title, contours)
+            cbar["max"] = np.amax(values[np.where(np.isfinite(values))])
+
+        # Treat variances if any
+        if input_data.has_variances:
+            variances = input_data.variances
+            # Check if dimensions of arrays agree, if not, plot the transpose
+            if (zlabs[0] == xlabs[0]) and (zlabs[1] == ylabs[0]):
+                variances = variances.T
+            # Apply colorbar parameters
+            if cbar["log"]:
+                with np.errstate(invalid="ignore", divide="ignore"):
+                    variances = np.log10(variances)
+            if cbar["min_var"] is None:
+                cbar["min_var"] = np.amin(
+                    variances[np.where(np.isfinite(variances))])
+            if cbar["max_var"] is None:
+                cbar["max_var"] = np.amax(
+                    variances[np.where(np.isfinite(variances))])
+        else:
+            variances = None
+
+        imview = ImageViewer(xe, xc, ye, yc, values, variances, resolution,
+                             cbar, plot_type, title, contours)
+
         for key, val in layout.items():
-            imview.fig.layout[key] = val
+            imview.fig_val.layout[key] = val
+            if variances is not None:
+                imview.fig_var.layout[key] = val
+
         if filename is not None:
-            write_image(fig=imview.fig, file=filename)
+            write_image(fig=imview.fig_val, file=filename)
```

Comment:
As i understand it is possible to plot the variances say as part of a notebook, but there is no way to save them out?